arch/arm/src/stm32wl5/hardware/stm32wl5_exti.h

```diff
@@ -103,4 +108,84 @@
 #define EXTI2_RADIOBSY     (1 << 14)  /* EXTI line 45: Radio busy wakeup */
 #define EXTI2_CDBGPWRUPREQ (1 << 15)  /* EXTI line 46: Debug power-up request wakup */
 
+/* Rising Trigger selection register */
+
+#define EXTI_RTSR1_BIT(n)  (1 << n)   /* 1=Rising trigger enabled (for Event and Interrupt) for input line */
```

Comment:
```suggestion
#define EXTI_RTSR1_BIT(n)  (1 << (n))   /* 1=Rising trigger enabled (for Event and Interrupt) for input line */
```
here and similar palces